compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -267,18 +267,6 @@ pub struct CoroutineInfo<'tcx> {
     /// Coroutine drop glue. This field is populated after the state transform pass.
     pub coroutine_drop: Option<Body<'tcx>>,
```

Comment:
Can we use the same device for drop body?

Reply:
Probably. I could investigate that as a follow-up.